src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -978,12 +978,17 @@ public static double Min(double val1, double val2)
             // This matches the IEEE 754:2019 `minimum` function
             //
             // It propagates NaN inputs back to the caller and
-            // otherwise returns the larger of the inputs. It
-            // treats +0 as larger than -0 as per the specification.
+            // otherwise returns the lesser of the inputs. It
+            // treats +0 as lesser than -0 as per the specification.
```

Comment:
I guess that this comment should be inverted like this for the `Min` variants:

```suggestion
            // treats -0 as lesser than +0 as per the specification.
```

Sorry for not spotting this earlier on the original PR!